spec/config/mongoid.yml

```diff
@@ -40,6 +40,8 @@ test:
     app_name: 'testing'
     background_indexing: false
     join_contexts: false
+  driver_options:
+    broken_view_options: false
```

Comment:
Can you please add a comment here that `validate_update_replace` shouldn't be given, because if someone gave it here as false it would make the test not do anything.

Reply:
done